packages/paste-core/components/card/package.json

```diff
@@ -28,10 +28,12 @@
     "type-check": "tsc --noEmit"
   },
   "peerDependencies": {
+    "@emotion/styled": "^10.0.10",
     "@twilio-paste/box": "^2.1.8",
     "@twilio-paste/style-props": "^0.1.7",
-    "react": ">= 16.0.0",
-    "react-dom": ">= 16.0.0"
```

Comment:
We should give everything prop-types too.  Eventually we will add them back to these packages if they don't already have them, and we might miss this unless we find a more permanent solution.

Reply:
OK, that's a good point. Lemme do some copy pasta and gitting and I'll push them up soon, might as well get it all done in one go